app/views/schools/user_tariffs/_tariff.html.erb

```diff
@@ -4,51 +4,49 @@
       <h5 class="mb-0">
         <a class="nav-link collapsed" data-toggle="collapse" data-target="#collapse<%= user_tariff.id %>" href="#collapse<%= user_tariff.id %>" role="button" aria-expanded="false" aria-controls="multiCollapseExample1">
           <%= user_tariff_title(user_tariff, true) %>
-          <%= button_tag 'View', class: 'btn right user-tariff-show-button' %>
+          <%= button_tag t('common.labels.view'), class: 'btn right user-tariff-show-button' %>
         </a>
       </h5>
     </div>
     <div id="collapse<%= user_tariff.id %>" class="collapse" aria-labelledby="heading<%= user_tariff.id %>" data-parent="#accordion<%= user_tariff.id %>">
       <div class="card-body">
         <table class="table table-charges">
           <tbody>
-          <tr>
-            <td class="description">Type</td>
-            <td class="value"><%= user_tariff.flat_rate ? 'Simple tariff' : 'Day/night tariff' %></td>
-          </tr>
-          <tr>
-            <td class="description">Start date</td>
-            <td class="value"><%= user_tariff.start_date.to_s(:es_compact) %></td>
-          </tr>
-          <tr>
-            <td class="description">End date</td>
-            <td class="value"><%= user_tariff.end_date.to_s(:es_compact) %></td>
-          </tr>
-
-          <tr>
-            <% if user_tariff.user_tariff_prices.any? %>
-              <td class="description">Consumption charges</td>
-              <td class="value nowrap">
-                <% user_tariff.user_tariff_prices.each do |price| %>
-                  <%= "#{price.start_time.to_s(:time)} to #{price.end_time.to_s(:time)} : " if !user_tariff.flat_rate %>
-                  <%= number_to_currency(user_tariff.user_tariff_prices.first.value, unit: '£') %> per kWh
-                  <br/>
-                <% end %>
-              </td>
-            <% else %>
-              <td colspan='10'>No consumption charges have been set yet</td>
-            <% end %>
-          </tr>
-
+            <tr>
+              <td class="description"><%= t('schools.user_tariffs.tariff_partial.type') %></td>
```

Comment:
Just another question about conventions - tariff_partial - above in the meters partial, you've not used meters_partial, just meters. I might have missed something though